avro-converter/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -1352,6 +1363,10 @@ private static boolean isEnumSchema(Schema schema) {
            schema.name() != null && schema.name().equals(AVRO_TYPE_ENUM);
   }
 
+  private static boolean isInstanceOfAvroEnum(Object value) {
```

Comment:
why factor this out? it seems better to have the `instanceof` check close to the cast.